lib/workers/bin/run_single_worker.rb

```diff
@@ -57,14 +57,22 @@
 worker_class = worker_class.constantize
 worker_class.before_fork
```

Comment:
I think previously, the `MiqUiWorker` called this to initialize the `UiConstants`, but I think that was the only use of `before_fork`.  There was some other `drb` related stuff in there as well, but I don't know that is needed when using the runner script.